### DISCUSSION TITLE
Make the background transparent

### DISCUSSION THREAD

**Discussion by lv2lrn4lf** [4ian/GDevelop#3900] [Archive] [answered]
how to make the background transparent? As in see through the scene background through the window.. so if i have a preview running how to make it be where i can see through the preview background so i can see other windows behind it while looking through it.

**Comment by lv2lrn4lf** [comment, marked as answer]
I found it, pixi.js is the file you want to alter. keyword to search is clear there are double locations of the same code so be careful, it's not as deep as the second one is, it'll be the first result that come when you first search for it.

**Reply by Silver-Streak** [reply]
Hello lv2lrn4lf!

The GDevelop project's github discussions section is more designated for longform discussion by the engine contributors and others around potential feature or codebase enhancements, or other platform bugfixes.

If you have questions on how to use the engine, please keep those to the forum or the discord. Thank you.
